dev/integration_tests/hybrid_android_views/android/app/src/main/java/io/flutter/integration/androidviews/MainActivity.java

```diff
@@ -36,6 +41,72 @@ private View getFlutterView() {
         return ((ViewGroup)root.getChildAt(0)).getChildAt(0);
     }
 
+    private String getViewName(View view) {
+        if (view instanceof FlutterImageView) {
+            return "FlutterImageView";
+        }
+        if (view instanceof FlutterSurfaceView) {
+            return "FlutterSurfaceView";
+        }
+        if (view instanceof FlutterTextureView) {
+            return "FlutterTextureView";
+        }
+        if (view instanceof FlutterView) {
+            return "FlutterView";
+        }
+        if (view instanceof ViewGroup) {
+            return "ViewGroup";
+        }
+        return "View";
+    }
+
+    private void getViewHierarchy(View current, String padding, StringBuilder builder) {
```

Comment:
As a recursive void method, the name `getViewHierarchy` seems confusing. How about `updateViewHierarchyString`?

Reply:
Done.